pkgs/applications/misc/kile-wl/default.nix

```diff
@@ -2,19 +2,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "kile-wl";
-  version = "unstable-2021-04-22";
+  version = "unstable-2021-06-01";
 
   src = fetchFromGitLab {
     owner = "snakedye";
     repo = "kile";
-    rev = "b97b9f1e5b33862b33918efaf23fd1c0c5d7058a";
-    sha256 = "sha256-97qJd3o8nJt8IX5tyGWtAmJsIv5Gcw1xoBFwxAqk7I8=";
+    rev = "5d8dcbdb908e1f8bdc37a731c6e1c5d8c2ace3ba";
+    sha256 = "sha256-f3S91Ui6SKcbkmlHqqjlx4M/Te+2no75W3eAVKd769s=";
   };
 
```

Comment:
```suggestion

nativeBuildInputs = [ scdoc ];
```

Reply:
and add `scdoc` to the arguments

Reply:
I've added that but nothing else is showing up in result/. Do I need to add a post install hook or something?

Reply:
yeah sure

Reply:
Ok, got that figured out, it should be installed to result/usr/share/man/man1 right?

Reply:
`result/share/man/man1`, drop the usr

Reply:
Gotcha

Reply:
Fixed in 4d5788b. Also updated the Cargo.lock.